shared/src/commonMain/kotlin/org/jetbrains/kotlinconf/APIClient.kt

```diff
@@ -97,7 +95,7 @@ class APIClient(
      * Vote for session.
      */
     suspend fun vote(sessionId: SessionId, score: Score?): Boolean {
-        if (userId == null) return false
+        userId ?: return false
```

Comment:
I personally find `if` check easier to read.
And also it works better with `if (userId != null) return true` in `ConferenceService::acceptPrivacyPolicy`, which cannot be shortened like this.

Reply:
Fair enough :)